readme.md

```diff
@@ -0,0 +1,120 @@
+[![Build Status](https://travis-ci.org/ninech/ordy.svg)](https://travis-ci.org/ninech/ordy)
+
+## Usage
+
+#### Ordering
+
+__Model usage__
+
+```rb
+class User < ApplicationRecord
+  include Ordy::Orm::ActiveRecord::Orderable
+  
+  has_many :comments
+
+  orderable_by do
+    # order by field
+    columns :name, :email
+    
+    # order by relations field
+    associations comments: :scripts
+    
+    # sort by specified order
+    specified(state: %w(new pendinguser migrating failed))
+    
+    # custom query
+    query :users do |scope, args|
+      scope.where(...).order(field: args[:direction])
+    end
+  end
+end
+```
+
+__Code usage__
+
+```rb
+# order by columns
+User.order_by(name: :asc)
+# or
+User.order_by('name-asc')
+
+# order by multiple columns
+User.order_by(name: :asc, email: :desc) 
+# or
+User.order_by(name: :asc).order_by(email: :desc)
+
+# order by association column
+User.order_by(user: :asc)
```

Comment:
This is not a valid example of ordering by association column. Keep the examples consistent.
In `Model` section you have `associations comments: :scripts`, so the example should be:

```rb
User.order_by(comments: :asc)
```